softlayer/vm/softlayer_vm.go

```diff
@@ -28,6 +28,7 @@ const (
 	softLayerVMtag = "SoftLayerVM"
 	ROOT_USER_NAME = "root"
 	deleteVMLogTag = "DeleteVM"
+	ISCSIMANAGEMENT = "ISCSI Management"
```

Comment:
Change this to: `ISCSI_MANAGEMENT` since that would read better and follows convention